presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -605,15 +606,15 @@ Avro Schema Evolution
 ---------------------
 
 Presto supports querying and manipulating Hive tables with Avro storage format which has the schema set
-based on an Avro schema file/literal. It is also possible to create tables in Presto which infers the schema
-from a valid Avro schema file located locally or remotely in HDFS/Web server.
+based on an Avro schema file/literal. It is also possible to create tables in Presto which infer the schema
```

Comment:
The grammar of this sentences was off in the first place. It sounds as if the ones that infer the schema are the tables, but it's Presto who does it.

Reply:
Reworded whole paragraph.